chef/cookbooks/glance/recipes/ceph.rb

```diff
@@ -24,35 +27,42 @@
 else
   # If external Ceph cluster will be used,
   # we need install ceph client packages
-  if node[:platform] == "suse" && File.exists?("/etc/ceph/ceph.client.admin.keyring")
+  return unless File.exists?(ceph_conf)
+
+  if node[:platform] == "suse"
     package "ceph-common"
-    package "python-ceph"
-  else 
-    return
   end
+
+  # In case when user will create store_user
+  # in ceph cluster manually and admin_keyring
+  # won't be present
+  return unless File.exists?(admin_keyring)
```

Comment:
Shouldn't we still try to access the ceph cluster with the user specified in the barclamp, to check it works?